app/views/shared/_provider_header_link.html.erb

```diff
@@ -1,4 +1,4 @@
-<ul id="navigation" class="govuk-header__navigation" aria-label="Top Level Navigation">
+<ul id="navigation" class="govuk-header__navigation no-print" aria-label="Top Level Navigation">
```

Comment:
Again, I think this element only appears within `app/views/layouts/_header.html.erb` and that this should be set as no print at that level.